Jenkinsfile.deploy

```diff
@@ -1,9 +1,21 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
 pipeline {
-    agent { label 'operator-slave' }
+    agent {
+        label 'kogito-operator-slave && !master'
+    }
+
+    // Needed for local build
+    tools {
+        maven 'kie-maven-3.6.2'
+        jdk 'kie-jdk11'
+    }
 
     parameters {
         string(name: 'DISPLAY_NAME', defaultValue: '', description: 'Setup a specific build display name')
+        
         string(name: 'BUILD_BRANCH_NAME', defaultValue: 'master', description: 'Which branch to build ? Set if you are not on a multibranch pipeline.')
+        string(name: 'GIT_AUTHOR', defaultValue: 'kiegroup', description: 'Which Git author repository ?')
```

Comment:
is it really necessary? can't this info be picked up automatically?

Reply:
nope we don't have that information. It is useful for anybody wanted to build on its own repository